packages/datagateway-common/src/detailsPanels/investigationDetailsPanel.component.tsx

```diff
@@ -49,15 +54,23 @@ const InvestigationDetailsPanel = (
           {t('investigations.details.start_date')}
         </Typography>
         <Typography>
-          <b>{investigationData.startDate}</b>
+          <b>
+            {investigationData.startDate
+              ? new Date(investigationData.startDate).toLocaleDateString()
```

Comment:
Is it better to use `date-fns` `format` function to format datestrings here rather than relying on `toLocaleDateString` to ensure a consistent format?